### DISCUSSION TITLE
Error when uploaded html files to git pages

### DISCUSSION THREAD

**Discussion by Amro-Salha** [community/community#44853] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, this is my first post on here so if it's in the wrong place; I'm sorry!

I'm currently in a coding bootcamp and was looking to get a headstart on creating my portfolio website. I made an HTML file, linked it to a CSS file, and had it extend from a base.html file. When I run the server on my local host it comes out exactly as I expect it to, however when I try to access it via git pages it comes out looking completely incorrect. Can someone please help me solve this? I will attach a link to my repository and if there are any other pertinent details needed please feel free to ask! I've already purchased the domain I want from google domains and want to resolve this so I can continue figuring out how to link the files to the domain.

https://github.com/Amro-Salha/portfolio

Thanks!

**Comment by airtower-luna** [comment]
Your site seems to be made with Django. That means it needs a server where the Python code can run (like you do when running it locally). That won't work on GitHub Pages, they support only static sites. You can use a static site generator (like Jekyll, Hugo, or even your own) to build the site via Actions, but the deployed site must be static HTML (possibly with CSS, scripts, etc.).
